### PR TITLE
Find homes for pages in documentation/wiki

### PR DESCRIPTION
**Pull request by jonasdn**
- Move coding-standard to contribute
- Remove toolbelt doc from wiki
- Move deck soldering guide to tutorials
- Remove Deviation Walkera controller from wiki
- Move taranis controller from wiki to tutorial
- Move pwm to thrust investigation to firmware repo
- Move bootloader arch from wiki to crazyflie2-stm-bootloader repo
- Remove bootloader client format
- Move hack-nrf guide from wiki to tutorials
- Move rpi as ground station to old tutorials
- Move rest of wiki to 'Old tutorials'

### CODE REVIEW

**Comment by knmcguire** [comment]
I'm checking this right now :) I'll come with comments soon!

File: src/documentation/tutorials/index.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ page_id: tutorials
 * {% id_link getting-started-buzzer-deck %}
 * {% id_link getting-started-z-ranger-deck %}
 * {% id_link getting-started-flow-deck %}
+* {% id_link solder-deck-pins %}
```
Comment (knmcguire) [inline src/documentation/tutorials/index.md:18]: Doubting if we should put this in these beginner tutorials at all... it voids warranty and we don't want it to appear to be the 'standard'.

We should remove it, as it is already referred to in https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#additional-information

File: src/documentation/wiki/bootloader-client-format.md [hunk 2]
```diff
@@ -1,53 +0,0 @@
----
```
Comment (knmcguire) [inline src/documentation/wiki/bootloader-client-format.md:1]: why was this removed? Shouldn't this go to cfclient firmware doc repo perhaps under the development header?

**Review by jonasdn — COMMENTED** [review]
Thanks!

I have pushed new commits!

File: src/documentation/tutorials/index.md (hunk 1 above)
Comment (knmcguire) [reply src/documentation/tutorials/index.md:18]: alright fair enough, but we should put at least a big fat warning on top about that we are not able to take back these products in case it is broken.

File: src/documentation/wiki/bootloader-client-format.md (hunk 2 above)
Comment (knmcguire) [reply src/documentation/wiki/bootloader-client-format.md:1]: aha misted that. no worries then!

File: src/documentation/tutorials/index.md [hunk 3]
```diff
@@ -30,7 +31,13 @@ page_id: tutorials
 ## Development
 
 * {% id_link getting-started-development %}
+* {% id_link hackrf-nrf %}
```
[the remark this answers is not in the document]
Comment (knmcguire) [reply src/documentation/tutorials/index.md:34]: I don't really see it as an tutorial but more as a regular guide. And talking about discover-ability, if I was developing for the NRF, I would expect this in the NRF doc and not in the tutorials

But we just leave this for now. What we could do to for more discover-ability for other guides in general, is to add more links here to other development guides that we now have within the repository documentation. Like the step-by-step guides and such.

**Comment by knmcguire** [comment]
alright! seems all resolved. let me just merge this and see if Jenkins is able to deploy it :)
